packages/api/src/controllers/reviews/v1/mod.rs

```diff
@@ -88,22 +97,47 @@ impl ReviewControllerV1 {
     pub async fn update(&self, id: i64, params: ReviewUpdateRequest) -> Result<Json<Review>> {
         tracing::debug!("update review: {:?} {:?}", id, params);
 
-        Ok(Json(Review::default()))
+        let fetched_review = sqlx::query_as!(
+            Review,
+            "UPDATE reviews SET name = $1, image = $2, review = $3 WHERE id = $4 RETURNING *",
+            params.name,
+            params.image,
+            params.review,
+            id
+        )
+        .fetch_one(&self.pool)
+        .await;
+
+        match fetched_review {
+            Ok(review) => Ok(Json(review)),
+            Err(_) => Err(ApiError::ApiCallError("Something went wrong".to_string())),
+        }
     }
 
     pub async fn find(
         &self,
         ReviewQuery { size, bookmark, .. }: ReviewQuery,
     ) -> Result<Json<ReviewGetResponse>> {
-        let _size = size;
+        let _size = size as i64;
         let _bookmark = bookmark;
 
         tracing::debug!("find query");
 
-        Ok(Json(ReviewGetResponse::Query(QueryResponse {
-            items: vec![],
-            total_count: 0,
-        })))
+       let found_reviews = sqlx::query_as!(
+            Review,
+            "SELECT * FROM reviews ORDER BY created_at DESC LIMIT $1",
+            _size
+        )
+        .fetch_all(&self.pool)
```

Comment:
```suggestion
       Review::query_builder().order_by_created_at().query().map(Review::from).fetch_all(&self.pool)
```

As I mentioned above, `Review` may have some relations. It means you need to JOIN query.